Packs/ThreatConnect/.pack-ignore

```diff
@@ -1,12 +1,18 @@
 [file:ThreatConnect_v2.yml]
 ignore=BA108,BA109,IN144,IN145
 
+[file:ThreatConnectV3.yml]
+ignore=IN124,IN121,IN126,IM111
```

Comment:
Why are you ignoring IN121 & IN126?
Why is IM111 both ignored for the YML and the PNG files?